<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Posição</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="ui_pb_check">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Verificar</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="ui_le_pos"/>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="ui_sp_time_margin"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Presidente</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Deputado Federal</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Governador</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="ui_le_dep_dis"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Tempo de margem</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="ui_le_senador2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Deputado distrital</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Senador 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="ui_le_senador1"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="ui_le_presidente"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ui_le_dep_fed"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="ui_le_governador"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Senador 1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ui_pb_file">
      <property name="text">
       <string>Carregar Resultado</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="ui_le_zeressima">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Data da zeressima</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ui_pb_file</tabstop>
  <tabstop>ui_le_zeressima</tabstop>
  <tabstop>ui_le_dep_dis</tabstop>
  <tabstop>ui_le_dep_fed</tabstop>
  <tabstop>ui_le_senador1</tabstop>
  <tabstop>ui_le_senador2</tabstop>
  <tabstop>ui_le_governador</tabstop>
  <tabstop>ui_le_presidente</tabstop>
  <tabstop>ui_le_pos</tabstop>
  <tabstop>ui_sp_time_margin</tabstop>
  <tabstop>ui_pb_check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
